packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -365,10 +365,6 @@ export const Popup: React.FC<PopupProps> &
         setPopupOpen(false, e);
         predefinedProps && _.invoke(predefinedProps, 'onMouseLeave', e, contentProps);
       };
-      contentHandlerProps.onClick = (e, contentProps) => {
-        setPopupOpen(true, e);
-        predefinedProps && _.invoke(predefinedProps, 'onClick', e, contentProps);
-      };
```

Comment:
I checked the original PR (https://github.com/microsoft/fluent-ui-react/pull/622) and this looks a bug as I don't see any scenario in which this is required as this handlers will go to `PopupContent`... Good catch 👍